nuagekubemon/policy/resourcemanager.go

```diff
@@ -136,8 +137,70 @@ func (rm *ResourceManager) GetPolicyGroupsForPod(podName string, podNs string) (
 	return &pgList, nil
 }
 
+const (
+	zoneAnnotationAdd    = "ZoneAnnotationAdd"
+	zoneAnnotationRemove = "ZoneAnnotationRemove"
+)
+
+func (rm *ResourceManager) createNuagePolicyUpdateForZoneAnnotation(namespace string, op string) error {
```

Comment:
Why are we calling this "update" when its mostly an add/delete as far as the namespace annotation is concerned.